ui/src/main/res/layout/tv_activity.xml

```diff
@@ -152,5 +152,19 @@
             app:iconTint="?attr/colorOnPrimary"
             app:layout_constraintBottom_toBottomOf="parent"
             app:layout_constraintStart_toStartOf="parent" />
+
+        <com.google.android.material.button.MaterialButton
+            android:id="@+id/menu_settings"
+            style="@style/Widget.MaterialComponents.Button"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_margin="16dp"
+            android:minWidth="0dp"
```

Comment:
```suggestion
```

Unnecessary

Reply:
This appears to be necessary or else the button is extra-long and the icon is off center. The other existing buttons also set the `minWidth` to 0

Reply:
That's because the style is wrong. `Widget.MaterialComponents.Button.TextButton.Icon` correctly  handles buttons that have a leading icon and trailing text, like this

![image](https://user-images.githubusercontent.com/13348378/95709227-d9d41e00-0c7b-11eb-9d18-17f205cef3bd.png)

Reply:
Ugh scratch that, there's no label here.